__mocks__/libs/react-native-batch.ts

```diff
@@ -21,3 +21,13 @@ export const BatchPush = {
 export const BatchMessaging = {
   setFontOverride: jest.fn(),
 }
+
+export const put = jest.fn()
+
+export const BatchEventData = jest.fn().mockImplementation(() => ({
+  addTag: jest.fn().mockReturnThis(),
+  putDate: jest.fn().mockReturnThis(),
+  putURL: jest.fn().mockReturnThis(),
+  put: put.mockReturnThis(),
+  _toInternalRepresentation: jest.fn().mockReturnValue({}),
+}))
```

Comment:
Je pense qu'il faut garder le faire que ça soit une classe, et que la fonction `put` ne fait qu'assigner une clé à une valeur dans un objet
```suggestion
export class BatchEventData {
  addTag() {
    return this
  }

  putDate() {
    return this
  }

  putURL() {
    return this
  }

  put(key: string, value: string) {
    return Object.assign(this, { [key]: value })
  }
}
```